doc/cheatsheet.md

```diff
@@ -0,0 +1,255 @@
+% djot cheatsheet
+
+## Basics
+
+| Markup | Result |
+| --- | --- |
+| `_italic_` or `{_italic_}` | *italic* |
+| `*bold*` or `{*bold*}` | **bold** |
+| `` `verbatim/code` `` | `verbatim/code` |
+| `H~2~O` | H<sub>2</sub>O |
+| `20^th^` | 20<sup>th</sup> |
+| `{=highlighted=}` | <mark>highlighted</mark> |
+| `{+insert+}` | <ins>insert</ins> |
+| `{-delete-}` | <del>delete</del> |
+
+Use triple-backticks for fenced verbatim blocks:
+
+~~~~~~
+```
+func say-hello(nm) {
+    print("hello ${nm}!");
+}
+```
+~~~~~~
+
+**Block quote**:
+
+~~~
+> Some blockquoted
+> content in here.
+~~~
+
+**Comments** `{% look like this`\
+`and can span multiple lines %}`.
+
+**Emojis**: `:+1:` → 👍, `:smiley:` → 😃 , etc.
+
+**Thematic break** (horizontal line): `***` or `---` on its own line.
+Can be longer than three characters, and may contain or be indented
+by spaces/tabs.
+
+
+
+## Math
+
+Use LaTeX formatting. `` $`p = mv` `` for inline math, and
+
+`` $$`E = K + U` ``
+
+for display math.
+
+
+## Special Characters
+
+  * `...` → … (ellipsis)
+  * `--` → – (an n-dash)
+  * `---` → — (an m-dash)
+  * Backslash-escape a punctuation character to remove
+    its special meaning.
+  * Backslash space = non-breaking space.
+  * Backslash at end of line = hard line break.
+
+## Links
+
+~~~
+<https://example.com>
+[read more](https://example.com)
+[read this too][foo bar]
+[one more link][]
+~~~
+
+then later:
+
+~~~
+[foo bar]: https://example.com
+[one more link]: https://example2.com
+~~~
+
+## Images
+
+~~~
+![beautiful skyline](clouds.jpg)
+![coastal shores][shore]
+![lush forests][]
+
+[shore]: the-beach.jpg
+[lush forests]: pines.jpg
+~~~
+
+## Headings
+
+`# H1`\
+`## H2`\
+`### H3`
+
+etc.
+
+## Lists
+
+~~~
+Itemized list:
+
+ * lions
+ * tigers
+ * bears
+~~~
+
+You can indent your list marker zero or more spaces, using same
+indent for items at same level. If you want a subsequent paragraph to
+be part of a list item, indent at least past the list item marker.
+Sublists must be preceded by a blank line, and must also be indented
+at least past the previous list item's marker.
+
+See the djot reference for the list of different types of markers
+allowed.
+
+Note that djot supports multi-line forms of almost everything,
+including list items.
```

Comment:
Technically what is going on here is "lazy paragraph wrapping."
I'm not sure I'd want to advertise it in the cheat sheet, because personally I think it's better not to use it.
But maybe there's another way to describe it?

Reply:
Good point. I removed the text, but also slightly expanded the example. I think the sample sets a good example of how one might format their multi-line list items.